### PR TITLE
Manage users for admin account

### PR DESCRIPTION
**Pull request by VoTaiTri**
using AJAX to search and sort users by name

### CODE REVIEW

**Comment by khuongnt1688** [comment]
ok

**Comment by NguyenHongViet** [comment]
Ôn rồi

File: app/models/user.rb [hunk 1]
```diff
@@ -23,4 +23,12 @@ class User < ActiveRecord::Base
   def is_admin?
     role == 'admin'
   end
+
+  def self.search search
```
Comment (dieunb) [inline app/models/user.rb]: move scope

File: app/models/user.rb [hunk 2]
```diff
@@ -23,4 +23,7 @@ class User < ActiveRecord::Base
   def is_admin?
     role == 'admin'
   end
+
+  scope :search_users, -> name {where('name LIKE ?', "%#{name}%")}
```
Comment (dieunb) [inline app/models/user.rb]: `search_by, ->name`

File: app/views/admin/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<div class="row">
+  <aside class="col-md-4">
+    <section class="user_info">
+      
+    </section>
+  </aside>
+  <div class="col-md-8">
+    <div class="col-md-12">
+      <div class="col-md-4">
+        <img id="book-image" src="../assets/<%= @user.avatar %>" />
```
Comment (dieunb) [inline app/views/admin/users/show.html.erb]: should use `image_tag`

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -13,9 +13,12 @@
     <div class="container">
       <% if notice %>
```
Comment (dieunb) [inline app/views/layouts/application.html.erb]: remove 
```
       <% if notice %>
         <p class="alert alert-success"><%= notice %></p>
+      <% elsif alert %>
         <p class="alert alert-danger"><%= alert %></p>
+      <% elsif flash.any? %>
       <% end %>
```

**Pull request merged**